speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/raceselectmenu.cpp

```diff
@@ -112,6 +91,33 @@ rmOnSelectRaceMan(void *pvRaceManTypeIndex)
 	}
 }
 
+/* Called when the menu is activated */
+static void
+rmOnActivate(void* dummy)
+{
+	GfLogTrace("Entering Race Mode Select menu\n");
+
+	LmRaceEngine().reset();
+
+	// SIMULATED DRIVING ASSISTANCE: skip this menu when activated
+	rmOnSelectRaceMan();
+}
+
+
+/* Exit from Race engine */
+static void
+rmOnRaceSelectShutdown(void* prevMenu)
+{
+	GfuiScreenActivate(prevMenu);
+
+	LmRaceEngine().cleanup();
+
+	LegacyMenu::self().shutdownGraphics(/*bUnloadModule=*/true);
+}
+
+
+
+
```

Comment:
- remove excess whitespace
- missing change comments
- functions we added to SD need to follow our guidelines (just SELECT this and run clang-format, do NOT run it on the whole file)

Reply:
in only added line 104, dont know why it says i changed more